pages/linux/skate.md

```diff
@@ -0,0 +1,36 @@
+# skate
+
+> Skate is simple and powerful key-value store. Skate can be used to save and retrieve anything you’d like—even binary data. It’s fully encrypted, backed up to the cloud (that you can self-host if you want) and can be synced with all your machines.
```

Comment:
```suggestion
> Simple and powerful key-value store.
```
I think this would be better

Reply:
yes, for the long description there's always `man`